client/src/components/NavBar/NavBar.tsx

```diff
@@ -6,18 +7,29 @@ import CircularProgress from '@material-ui/core/CircularProgress';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
 import { useHistory } from 'react-router-dom';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import LoggedInBar from './AuthBars/LoggedInBar';
 import LoggedOutBar from './AuthBars/LoggedOutBar';
 import CssBaseline from '@material-ui/core/CssBaseline';
 
-const NavBar = (): JSX.Element => {
-  const classes = useStyles();
+interface Props {
+  children?: React.ReactNode;
+}
 
+const NavBar = ({ children }: Props): JSX.Element => {
+  const [isLandingPage, setIsLandingPage] = useState<boolean>(false);
+  const classes = useStyles();
   const { loggedInUser } = useAuth();
   const { initSocket } = useSocket();
```

Comment:
You can remove this line of code and the useEffect below which initializes the socket, we don't want to initialize the socket within the Navbar :)

Reply:
Yes, it's true. Done